torch/package/package_exporter.py

```diff
@@ -929,10 +1001,25 @@ def _execute_dependency_graph(self):
                 if attrs.get("is_pickle") is True:
                     # This node came from save_source_pickle, we don't need to write any source for it.
                     continue
-
                 is_package = attrs["is_package"]
                 source = attrs["source"]
                 self._write_source_string(module_name, source, is_package)
+            elif action == _ModuleProviderAction.SELECTIVE_EXTERN:
+                # pdb.set_trace()
+                is_package = attrs["is_package"]
+                source = attrs["source"]
+                # pdb.set_trace()
+                if is_package and source:
```

Comment:
why this if statement? Under what conditions would these values  be `False-`y, and should we handle them?